### PR TITLE
Always use Release Dll for assimp

### PR DESCRIPTION
**Pull request by michielvandencamp-prompto**
When QA tests a non-shipping build they get errors as they do not have the required dev dependencies
We never test or debug the assimp DLL itself so we'd rather always use the release version

### CODE REVIEW

File: Engine/Plugins/Marketplace/RuntimeMeshImportExport/Source/RuntimeMeshImportExport/Private/RuntimeMeshImportExport.cpp [hunk 1]
```diff
@@ -16,11 +16,11 @@ void FRuntimeMeshImportExportModule::StartupModule()
 	// This code will execute after your module is loaded into memory; the exact timing is specified in the .uplugin file per-module
 	FString PluginBaseDir = IPluginManager::Get().FindPlugin("RuntimeMeshImportExport")->GetBaseDir();
 	FString configString;
-#if UE_BUILD_SHIPPING
+
+	//AMCHANGE_begin
+	//#AMCHANGE Always use the release DLLs, not all testers and QA always have all the dev dependencies
```
Comment (etiennebeaucheminp-prompto) [inline Engine/Plugins/Marketplace/RuntimeMeshImportExport/Source/RuntimeMeshImportExport/Private/RuntimeMeshImportExport.cpp]: `dev dependencies` is not a very clear way to describe it.

I would propose: `not all testers and QA always have the debug dll's required to load the debug assimp dll.`

**Review by etiennebeaucheminp-prompto — COMMENTED** [review]
Can you also remove the debug `.lib` as well? It is located in `assimp/lib/...`

**Review by etiennebeaucheminp-prompto — CHANGES_REQUESTED** [review]
I just found out that you need to also change the `RuntimeMeshImportExport.Build.cs` because it will still try to build with the debug dll and lib.

Look at `RuntimeMeshImportExport.Build.cs line 95`

**Comment by michielvandencamp-prompto** [comment]
> 
> 
> I just found out that you need to also change the `RuntimeMeshImportExport.Build.cs` because it will still try to build with the debug dll and lib.
> 
> Look at `RuntimeMeshImportExport.Build.cs line 95`

Good catch, should be fixed now

**Review by etiennebeaucheminp-prompto — APPROVED** [review]

**Pull request merged**